cmd/run.go

```diff
@@ -76,6 +76,14 @@ var RunCommand = cli.Command{
 					Name:  "test-param, p",
 					Usage: "provide a test parameter",
 				},
+				cli.BoolFlag{
```

Comment:
This is already implemented, but we have those flags only on `testground run composition`, and it is handy to have them under `testground run single` as well.